docs/shared/getting-started/nx-setup.md

```diff
@@ -2,14 +2,18 @@
 
 ## Set up a New Nx Workspace
 
-Run the following command to create a new workspace.
+> (placeholder) explain what a "workspace" is or link to a glossary of terms we use?
+
+Create a new Nx workspace by running the follow command and follow the prompts
 
 ```bash
 # pass @latest in case npx cached an older version of create-nx-workspace
 npx create-nx-workspace@latest
 ```
 
-When creating a workspace, you will have to choose a preset, which will preconfigure a few things for you.
+Nx provides a handful of presets to assist in organizing your workspace. You can choose which ever makes the most sense for your project.
+
+(Placeholder) learn more about presets and why would want to use them.
```

Comment:
should we talk about presets here? link out to the presets we have? it's going pose the questions
- what is preset?
- which preset do I use?
- do I have to use it?